rgdc/rgdc/utils.py

```diff
@@ -79,13 +79,14 @@ def datetime_to_str(value: object):
     return value
 
 
-def download_checksum_file_to_path(file: Dict, path: Path):
+def download_checksum_file_to_path(file: Dict, path: Path, overwrite: bool):
```

Comment:
when passing a kw though multiple functions, I think it's best practice to only have a default on the outermost caller, to avoid confusion. In this case the caller's default is `True` so it'd be doubly weird.

Reply:
I guess I'm okay with leaving `overwrite` as a required argument for now, although it seems like it should have a sensible default. Come to think of it, I kind of think that `overwrite` should become `keep_existing` or something (with the default to `True`), to better convey the behavior. This isn't really a big deal though.

Reply:
I like `keep_existing`.